modin/core/dataframe/pandas/dataframe/utils.py

```diff
@@ -59,10 +59,12 @@ def build_sort_functions(
         A named tuple containing the functions to pick quantiles, choose pivot points, and split
         partitions for sorting.
     """
+    frame_len = len(modin_frame.index)
+    is_column_numeric = pandas.api.types.is_numeric_dtype(modin_frame.dtypes[column])
```

Comment:
moved this out of the kernel functions so we wouldn't need to broadcast the `modin_frame` to workers